library/include/rocblas-auxiliary.h

```diff
@@ -14,80 +13,64 @@
  * \brief rocblas-auxiliary.h provides auxilary functions in rocblas
 */
 
-/* ============================================================================================ */
-/*! \brief  indicates whether the pointer is on the host or device. currently HIP API can only
-recoginize the input ptr on deive or not
-can not recoginize it is on host or not */
-ROCBLAS_EXPORT rocblas_pointer_mode rocblas_pointer_to_mode(void* ptr);
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-/********************************************************************************
- * \brief rocblas_handle is a structure holding the rocblas library context.
- * It must be initialized using rocblas_create_handle()
- * and the returned handle must be passed
```

Comment:
Are we loosing some detail here?

Reply:
This information is in the file   library/include/rocblas-types.h  .  I think it is better to have the information in library/include/rocblas-types.h ,  and not have it duplicated in both files.